app/styles/_overview.less

```diff
@@ -43,12 +43,11 @@
   margin-bottom: 5px;
   margin-top: 5px;
   padding: 5px;
-  .truncate();
   .mini-log-content {
     line-height: @mini-log-line-height;
     min-height: (@mini-log-num-lines * @mini-log-line-height);
     .mini-log-line {
-      white-space: pre;
```

Comment:
I incorrectly assumed that would break .truncate()'s [.text-overflow()](https://github.com/twbs/bootstrap/blob/master/less/mixins/text-overflow.less#L7).  It looks like it doesn't, so I've restored it.